extensions/applicationinsights-debugplugin-js/README.md

```diff
@@ -0,0 +1,188 @@
+# Microsoft Application Insights JavaScript SDK - Debug Plugin
+
+[![Build Status](https://travis-ci.org/microsoft/ApplicationInsights-JS.svg?branch=master)](https://travis-ci.org/microsoft/ApplicationInsights-JS)
+[![npm version](https://badge.fury.io/js/%40microsoft%2Fapplicationinsights-debugplugin-js.svg)](https://badge.fury.io/js/%40microsoft%2Fapplicationinsights-debugplugin-js)
+<!-- [![gzip size](https://img.badgesize.io/https://az416426.vo.msecnd.net/scripts/b/ai.2.min.js.svg?compression=gzip&softmax=30000&max=35000)](https://img.badgesize.io/https://az416426.vo.msecnd.net/scripts/b/ai.2.min.js.svg?compression=gzip&softmax=30000&max=35000) -->
+<!-- [![minified size size](https://img.badgesize.io/https://az416426.vo.msecnd.net/scripts/b/ai.2.min.js.svg?label=minified%20size)](https://img.badgesize.io/https://az416426.vo.msecnd.net/scripts/b/ai.2.min.js.svg?label=minified%20size) -->
+
+
+This README is broken down into several parts:
+
+- **Supported Browsers**
+- **Who** should use this plugin
+- **What** this plugin does
+- **When** to use this plugin
+- **Where** to get more information
+- **Why** this plugin exists
+- **Getting Started**
+
+these parts go together to explain the reasoning for this plugin's existance and provides
+an explaination as to how everything works. Each of these sections will finish with a
+bullet-point list that summarizes the section
+
+## Supported Browsers
```

Comment:
Add supported browsers at the bottom of the list, other sections seem more important

Reply:
+1